src/components/SearchBox.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+
+import PropTypes from 'prop-types';
+
+export const SearchBox = props => {
+  const [query, setQuery] = useState('');
+  return (
+    <form
+      onSubmit={e => {
+        e.preventDefault();
+        props.onSubmit(query);
+        setQuery('');
+      }}
+    >
+      <label>
+        Start by searching for a phone number
+        <input
+          type="text"
```

Comment:
It should be either `type="search"` or `type="tel"`. Probably search is better. :)